lib/cifar10.ex

```diff
@@ -36,6 +40,9 @@ defmodule Scidata.CIFAR10 do
     * `:transform_labels` - similar to `:transform_images` but applied to
       dataset labels
 
+    * `:test_set` - indicate whether the training set or the test set
+        should be fetched
```

Comment:
This (and below) is indented too much. See the other options above. :)

Reply:
I think I've fixed it